api/tests/integration/audit/test_audit_logs.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import reverse
+from rest_framework import status
+
+
+def test_audit_logs_only_makes_two_queries(
+    admin_client, project, environment, django_assert_num_queries
+):
+    url = reverse("api-v1:audit-list")
+
+    with django_assert_num_queries(2):
+        res = admin_client.get(url, {"project": project})
+
+    assert res.status_code == status.HTTP_200_OK
```

Comment:
Would probably be good here to check if there is any data returned. I can't remember what triggers an audit log message. Maybe creating the environment will trigger one but I'm not 100%. Could we just verify here that at least one audit log resource is returned?